### PR TITLE
feat(message-system, coinjoin):enabled coinjoin from 23.5, removed general banner, and returned feedback form

### PR DESCRIPTION
**Pull request by hynek-jina**
- Coinjoin enabled from `23.5.1`
- Removed general high fee warning banner: ~~`Sending and receiving of bitcoin is delayed due to high network activity`~~
- Returned banner in coinjoin account to: `Coinjoin is in public beta. Your feedback is appreciated` and button `Give Feedback`

### CODE REVIEW

File: packages/message-system/src/config/config.v1.json [hunk 1]
```diff
@@ -100,10 +141,7 @@
                     {
                         "domain": "coinjoin",
                         "flag": false,
-                        "isPublic": true,
-                        "averageAnonymityGainPerRound": 0.34,
-                        "roundsFailRateBuffer": 10,
-                        "roundsDurationInHours": 3
+                        "isPublic": true
```
Comment (matejkriz) [inline packages/message-system/src/config/config.v1.json:152]: content of this message should be updated too...

Comment (hynek-jina) [reply packages/message-system/src/config/config.v1.json:152]: Params are on lines 109-111.. Or do you mean something else?

File: packages/message-system/src/config/config.v1.json [hunk 2]
```diff
@@ -72,6 +46,73 @@
...
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=23.5.1",
+                        "mobile": "!",
+                        "web": ">=23.5.1"
+                    }
+                }
+            ],
+            "message": {
```
Comment (matejkriz) [inline packages/message-system/src/config/config.v1.json:98]: validation failed, message has to have `content` https://github.com/trezor/trezor-suite/actions/runs/4960816845/jobs/8876813361?pr=8450#step:11:13

File: packages/message-system/src/config/config.v1.json (hunk 1 above)
Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:152]: No, I mean rewriting the content `Coinjoin is temporarily unavailable due to technical issues` to something meaning "Update to latest Suite version to use coinjoin"

Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:152]: bfc55e552d0b9f0bffd3984941ca5d1a29c4bf76

File: packages/message-system/src/config/config.v1.json (hunk 2 above)
Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:98]: bfc55e552d0b9f0bffd3984941ca5d1a29c4bf76

File: packages/message-system/src/config/config.v1.json (hunk 1 above)
Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:152]: anyway, I merged this message with the other one that disable coinjoin for older Suite versions, there is no need to define `isPublic` explicitly, we don't need that param anymore, it was just before public lauch of coinjoin...

Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:152]: my bad, should be fixed now

Comment (matejkriz) [reply packages/message-system/src/config/config.v1.json:152]: e9edd2d8cc1f0ed17ffc1850eb59c3573ec64667

**Review by tomasklim — APPROVED** [review]
Okay from my side. Does it mean that we will enable coinjoin for users (non-eap) next monday?

**Comment by hynek-jina** [comment]
> Okay from my side. Does it mean that we will enable coinjoin for users (non-eap) next monday?

Yes

**Pull request merged**